api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const ncNews = axios.create(
+    { baseURL: "https://nc-news-8ork.onrender.com/api"}
+)
+
+export const getArticles = () => {
+    return ncNews.get('/articles').then(({ data }) => {
```

Comment:
Great job abstracting this away from the component! Just a small thing, you might not need the .then here 🙂